bin/kubectl-rabbitmq

```diff
@@ -382,18 +408,11 @@ main() {
     "help")
         usage
         ;;
-    "--help")
-        usage
-        ;;
-    "-h")
-        usage
-        ;;
     "version")
         version
         ;;
     *)
         usage
-        exit 1
```

Comment:
Why are we removing exit code 1 on error?

Reply:
It's back